runtime/kusama/src/lib.rs

```diff
@@ -1732,7 +1732,7 @@ sp_api::impl_runtime_apis! {
 			Err(mmr::Error::PalletNotIncluded)
 		}
 
-		fn generate_batch_proof(_leaf_indices: Vec<u64>)
+		fn generate_batch_proof(_block_numbers: Vec<BlockNumber>)
```

Comment:
You also need to change `fn generate_historical_batch_proof()` signature (for all runtimes)